corehq/motech/value_source.py

```diff
@@ -245,4 +245,12 @@ def _recurse_form_questions(form_dict, path, result_):
     _recurse_form_questions(form_json['form'], [b'/data'], result)  # "/data" is just convention, hopefully
     # familiar from form builder. The form's data will usually be immediately under "form_json['form']" but not
     # necessarily. If this causes problems we may need a more reliable way to get to it.
+
+    if 'meta' in form_json['form'] and 'received_on' in form_json:
```

Comment:
Yeah, I thought of break this up more. But I decided against it for the sake of simplicity: I think all real form submissions would have all of these. This commit was so that this function can also be used for testing and sample data that doesn't come with metadata. But I'd be happy to break it up if you think it would be more useful.

Reply:
:thinking: You convinced me. :grin:

Reply:
haha, cool!